docs/userguide/configure-controllers.md

```diff
@@ -48,9 +48,9 @@ E.g. Specify a controller list:
 --controllers=cluster,clusterStatus,binding,xxx
 ```
 
-E.g. Disable some controllers:
+E.g. Disable some controllers and keep other default controllers turned on:
```

Comment:
Fell good to me, /cc @RainbowMango @hanweisen

Reply:
Enable/Disable controller a bit counterintuitive, with this UT will save much time for new users

Reply:
Wow, I just realized that is me who made the mistake. Sorry for that.